backend/config/settings.py

```
@@ -0,0 +1,33 @@
+    test_client_db_name: str
+    test_client_db_user: str
+    test_client_db_password: str
+    test_client_db_port: int = 5432
+    test_client_db_type: str = "postgres"
+
+    # 로깅 설정
+    log_level: str = "INFO"
+    azure_appinsights_connection_string: Optional[str] = None
+    
+    # API 설정
+    api_host: str = "0.0.0.0"
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)